step-release-vis/src/app/components/environments/environments.ts

```diff
@@ -79,11 +83,10 @@ export class EnvironmentsComponent implements OnInit {
    * @param environments an array of environments
    */
   private processEnvironments(environments: Environment[]): void {
-    this.envWidth = window.innerWidth - this.ENV_RIGHT_MARGIN;
-    this.envHeight = window.innerHeight / this.ENVS_PER_PAGE;
-    this.timelinePointsAmount = Math.floor(
-      this.envWidth / this.TIMELINE_POINT_WIDTH
-    );
+    this.envSmallWidth = window.innerWidth - this.ENV_RIGHT_MARGIN;
+    this.envSmallHeight = window.innerHeight / this.ENVS_PER_PAGE;
+    this.envBigWidth = window.innerWidth - this.ENV_RIGHT_MARGIN;
```

Comment:
why do we have 2 of these? it is the same as the small one.. lets just use envW ;)

Reply:
I change that behaviour in the next pr
https://github.com/googleinterns/step251-2020/blob/b9a59533f4d63e8625ac4250f0def0a11065b5ab/step-release-vis/src/app/components/environments/environments.ts#L87-L91